lib/npg_tracking/Schema/Result/Instrument.pm

```diff
@@ -410,12 +395,11 @@ Change the instrument_status for an instrument. This means adding a new row to
 the instrument_status table and setting the iscurrent field to 1, after
 marking all the previous rows for the isntrument as iscurrent 0.
 
-Two arguments are required, an instrument status and a user in that order. In
-both cases the primary key can be supplied, or the (case-insensitive)
-description or username fields respectively.
+Two arguments are required, an instrument status description and a user id or
+name.
 
-    $instrument_result_object->update_instrument_status( 4, 'jo3' );
-    $instrument_result_object->update_instrument_status( 'Down', 5 );
+    $instrument_result_object->update_instrument_status( 'Down', 'jo3');
+    $instrument_result_object->update_instrument_status( 'Down', '5);
 
 =cut
 
```

Comment:
I think status_description (instead of status_identifier) is better name for the parameter for this function.